mysql-test/main/func_sformat.test

```diff
@@ -250,6 +250,14 @@ set @a=3.14;
 select sformat('{:f}', @a);
 set names latin1;
 
+echo #;
+echo # MDEV-29646 sformat('Num [{:20}]', 42) gives incorrect result in view;
+echo #;
+
+create view v1 as select sformat('Num [{:20}]', 42);
+select * from v1;
+drop view v1;
+
```

Comment:
good basic test, but probably more test cases are needed. Also this is in 10.8 so its after the "End of 10.7". Adding a "Start of 10.8" will help merging eventually.